src/wasm.rs

```diff
@@ -971,7 +977,12 @@ impl WasmModule {
                             mutable: global_ty.mutable,
                         });
                     }
-                    e => panic!("Have not implemented import section entry type {:?}", e),
+                    ImportSectionEntryType::Memory(memory_ty) => {
+                        self.memories.push(*memory_ty);
+                    }
+                    ImportSectionEntryType::Table(table_ty) => {
+                        self.tables.push(*table_ty);
+                    }
```

Comment:
Kinda nervous about this, as we don't actually support additional memories or tables. I suppose I can't think of anywhere else in the code we assume this (or at least nowhere that'd cause subtle unsafety)

Reply:
I believe these were needed because clang was emitting empty imports. They were empty, but aWsm was failing to parse the empty stubs.

Though I will admit there was a bit of poking-the-code-until-it-works here.

Reply:
It does match the `process_memory_section` function's behavior:
https://github.com/gwsystems/aWsm/blob/59e68b4dd1c1a9fb19df1c09f2301d0407302ac9/src/wasm.rs#L938